pkg/hubble/exporter/exporter_test.go

```diff
@@ -0,0 +1,64 @@
+// Copyright 2021 Authors of Cilium
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// +build !privileged_tests
+
+package exporter
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"testing"
+
+	observerAPI "github.com/cilium/cilium/api/v1/observer"
+	v1 "github.com/cilium/cilium/pkg/hubble/api/v1"
+	nodeTypes "github.com/cilium/cilium/pkg/node/types"
+	"github.com/golang/protobuf/ptypes/timestamp"
+
+	"github.com/sirupsen/logrus"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestExporter(t *testing.T) {
+	// override node name for unit test.
+	nodeName := nodeTypes.GetName()
+	newNodeName := "my-node"
+	nodeTypes.SetName(newNodeName)
+	defer func() {
+		nodeTypes.SetName(nodeName)
+	}()
+	events := []*v1.Event{
```

Comment:
Could we have tests for the various event types? Notably `Flow`, `LostEvent`, `AgentEvent`, `DebugEvent`?
One could also test `EventToGetFlowsResponse()` in a separate test function.

Reply:
added some more test cases.